solidity/contracts/BondedECDSAKeep.sol

```diff
@@ -1,3 +1,4 @@
+/* solium-disable function-order */
```

Comment:
Any particular reason for this? I would kind of rather just go with the function order in our linter. Fewer toggles is better.

Reply:
We tried to place the functions in a logical order as they may be used in the flow. But yeah, sure I can rearrange the functions order to satisfy solium.

Reply:
I actually get this and it tends to be my first preference during initial dev, but there are some strong arguments for function ordering for visibility. Diligence also flagged this possibility. My main thing is I'd rather either go all in on ignoring this, or not ignore it ever heh. I'm already twitchy over the other exceptions we have to solium.

Reply:
As I like the idea to conform to the rules and order the functions accordingly, there is one place where IMO it would be better to disable the check. 
It's in `BondedECDSAKeepVendor` contract where we have a long list of getters/setters pairs where getters are public and setters internal:
https://github.com/keep-network/keep-ecdsa/blob/d7bf095e964580e40eac0c4bae5575232853ec9c/solidity/contracts/BondedECDSAKeepVendor.sol#L158-L260